src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityQuantumTank.java

```diff
@@ -593,6 +593,20 @@ private void setVoiding(boolean isPartialVoid) {
         }
     }
 
+    @Override
+    public ItemStack getPickItem(EntityPlayer player) {
+        if(!player.isCreative()) return super.getPickItem(player);
```

Comment:
Shouldn't this also require the player to be sneaking?

Reply:
shouldnt need to, at least based on vanilla behavior